models/MARBL_param_estimation/model_mod.f90

```diff
@@ -0,0 +1,591 @@
+! DART software - Copyright UCAR. This open source software is provided
```

Comment:
I thought we agreed that we will only do the joint state-parameter estimation one. I would stick to one marbl directory and one model mod for now. These could be consolidated into one in the future. I mean something that comes to mind now is to do no_copy_back  on the state variables to turn it into a parameter estimation only.

Reply:
Yes, we did agree to only do the joint state-parameter estimation model-mod. I had all three model-mods being developed on the same branch to make it easier to switch between them, which is why they're all in the same pull request; I apologize if that's bad form. In any case, `MARBL_joint_estimation` is the one that we agreed to pull into DART.